src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -204,4 +228,28 @@ private function hasProductWithName(array $products, string $name): bool
 
         return false;
     }
+
+    private function hasReviewsWithTitles($titles): bool
+    {
+        $productReviews = $this->responseChecker->getValue($this->client->getLastResponse(), 'reviews');
+
+        foreach ($titles as $title) {
+            if (!$this->hasKeyWithValue($productReviews, 'title', $title)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    private function hasKeyWithValue(array $array, string $key, string $value): bool
+    {
+        foreach ($array as $arrayValue) {
```

Comment:
in `resposneChecker` we have the function `hasKeyWithValue` is it the same case?